migrations/versions/969126bd800f_.py

```diff
@@ -5,6 +5,7 @@
 Create Date: 2018-01-31 15:20:30.396533
 
 """
+from __future__ import print_function
```

Comment:
It is useful because it prevents Python 2 developers from adding __print__ statements that break Python 3.  It is also required when commas, __end=__, and __file=__ are used.